### ISSUE TITLE
RFE Unmaintained units dialog should list unmaintained units.

### ISSUE THREAD

**Issue by pakfront** [MegaMek/mekhq#3152]
### Environment
13:28:29,662 INFO  [mekhq.MekHQ] {main}
mekhq.MekHQ.initializeLogging(MekHQ.java:290) - Starting MekHQ v0.49.7
	Release Date: 2022-02-18
	Today: 2022-03-01
	Origin Project: MekHQ
	Java Vendor: AdoptOpenJDK
	Java Version: 11.0.10
	Platform: Windows 10 10.0 (amd64)
	System Locale: en_US

### Description
Unmaintained units dialog should list unmaintained units. Otherwise very confusing for new users.

**Comment by Windchild292** [comment]
Are you referring to the nag?
If so, I purposefully didn't add it after testing with large campaigns, as it was causing graphical problems when there's a huge number of units and actually was a UX negative when I was trying it out.

Further, MekHQ has a unit type filter, colour, and a column for unmaintained units...

**Comment by pakfront** [comment]
Yes, I am referring to the nag. apologies for the lack of specifics.  Understandable if there are UI issues, but maybe adding some simple directions on how to maintain them, something like: "Unmaintained units can be assigned a tech crew in the Hangar."

**Comment by IllianiCBT** [comment]
With the nag update in 2024 this was resolved. Closing.
